src/app/refa/stages/[id]/page.tsx

```diff
@@ -25,91 +26,103 @@ export default async function Page({
-                </SheetTitle>
-                <SheetDescription className="text-left">
-                  <div className="my-4 flex">
-                    <SheetTitle>People</SheetTitle>
-                    <Badge
-                      variant={
-                        area.people === "High"
-                          ? "destructive"
-                          : area.technology === "Medium"
-                            ? "secondary"
-                            : "default"
-                      }
-                      className="ml-2"
-                    >
-                      {area.people}
-                    </Badge>
-                  </div>
-                  <ul>
-                    {area.people_practices.map((practice) => (
-                      <li key={practice}>{practice}</li>
-                    ))}
-                  </ul>
-                  <Separator className="my-4" />
-                  <div className="my-4 flex">
-                    <SheetTitle>Process</SheetTitle>
-                    <Badge
-                      variant={
-                        area.process === "High"
-                          ? "destructive"
-                          : area.technology === "Medium"
-                            ? "secondary"
-                            : "default"
-                      }
-                      className="ml-2"
-                    >
-                      {area.process}
-                    </Badge>
-                  </div>
-                  <ul>
-                    {area.process_practices.map((practice) => (
-                      <li key={practice}>{practice}</li>
-                    ))}
-                  </ul>
-                  <Separator className="my-4" />
-                  <div className="my-4 flex">
-                    <SheetTitle>Technology</SheetTitle>
-                    <Badge
-                      variant={
-                        area.technology === "High"
-                          ? "destructive"
-                          : area.technology === "Medium"
-                            ? "secondary"
-                            : "default"
-                      }
-                      className="ml-2"
-                    >
-                      {area.technology}
-                    </Badge>
-                  </div>
-                  <ul>
-                    {area.technology_practices.map((practice) => (
-                      <li key={practice}>{practice}</li>
-                    ))}
-                  </ul>
-                </SheetDescription>
-              </SheetHeader>
-            </SheetContent>
-          </Sheet>
+            />
+          </AreaSheet>
         ))}
       </div>
     </div>
   );
 }
+
+export function AreaSheet({
+  area,
+  children,
+}: Readonly<{
+  area: Area & { artefacts: Artefact[] };
+  children: React.ReactNode;
+}>) {
+  return (
+    <Sheet key={area.area_id}>
+      <SheetTrigger asChild>{children}</SheetTrigger>
```

Comment:
Could this area sheet have different sheet triggers and display different content (info or assessment) accordingly? Just because I also edited this file haha

Reply:
No need to use the Sheet from this file. You can just import a new <Sheet /> and use it for displaying the content you wish to display.

Reply:
@NingyueZhou please check the Shadcn documentation using Sheets.